jenkins-scripts/dsl/sdformat.dsl

```diff
@@ -1,10 +1,10 @@
 import _configs_.*
 import javaposse.jobdsl.dsl.Job
 
-def sdformat_supported_branches = [ 'sdformat6' , 'sdformat9', 'sdformat12' ]
-def sdformat_gz11_branches = [ 'sdformat9', 'sdformat10', 'sdformat12', 'main' ]
+def sdformat_supported_branches = [ 'sdformat6' , 'sdformat9', 'sdformat12', 'sdformat13' ]
```

Comment:
The branches are `sdf` instead of `sdformat` 🤔  This variable name is confusing

Reply:
yeah, that variable is poorly named; it's package versions not branch names. I'll rename it real quick

Reply:
refactoring in a separate PR: https://github.com/gazebo-tooling/release-tools/pull/789